tasks/eth/base-003-holocene-fp-upgrade/.env

```diff
@@ -0,0 +1,12 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+
+# Base L1 PAO
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
+# This is Base's safe but the Nested.just file uses council as a keyword and
+# we want to minimize changes
+COUNCIL_SAFE=0x9855054731540A48b28990B63DcF4f33d8AE46A1
+# Foundation Operations Safe
+FOUNDATION_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A
+
+# No nonce overrides needed, this is the next task for B1PAO, BOS, FOS
```

Comment:
We should hardcode the NONCE of the safes following our notion doc. and also update the nested validation with the storage increment. 
```
SAFE_NONCE=4
SAFE_NONCE_0X9855054731540A48B28990B63DCF4F33D8AE46A1=16
SAFE_NONCE_0X9BA6E03D8B90DE867373DB8CF1A58D2F7F006B3A=97
```

Reply:
There won't be increments/overrides in the validation, even if we set these explicitly. I'll add them anyways.